packages/nc-gui-v2/pages/signup/[[token]].vue

```diff
@@ -43,17 +44,23 @@ const formRules = {
     {
       validator: (_: unknown, v: string) => {
         return new Promise((resolve, reject) => {
-          if (isEmail(v)) return resolve(true)
+          if (!v?.length || isEmail(v)) return resolve(true)
           reject(new Error(t('msg.error.signUpRules.emailInvalid')))
         })
       },
       message: t('msg.error.signUpRules.emailInvalid'),
     },
   ],
   password: [
-    // Password is required
-    { required: true, message: t('msg.error.signUpRules.passwdRequired') },
```

Comment:
Why was the `required` rule removed?

Reply:
I have added a custom validator which checks for this as well.

Reply:
@bcakmakoglu

Reply:
Ok 😄